addons/utm/models/utm_mixin.py

```diff
@@ -59,3 +53,37 @@ def tracking_fields(self):
             ('utm_source', 'source_id', 'odoo_utm_source'),
             ('utm_medium', 'medium_id', 'odoo_utm_medium'),
         ]
+
+    def _generate_url_parameters(self):
```

Comment:
Not sure this method adds readability to the code ... did you change anything ?

Reply:
Now we use the identifier instead of the name in the URL, but I can move this code to the link tracker model and remove this method